src/components/flowchart/styles/flowchart.scss

```diff
@@ -0,0 +1,20 @@
+@import './node-icon';
+@import './node-text';
+@import './edges';
+@import './tooltip';
+
+.pipeline-flowchart {
+  height: 100%;
+}
+
+.pipeline-flowchart__graph {
+  display: block;
+}
+
+.node {
```

Comment:
Could node be too general a name? Might be worth introducing css modules or following your .pipeline-flowchart__node as a class name

Reply:
'node' and 'edge' are names I've used across the app for a specific type of element, so I don't think it's confusing at all - it is after all, a graph with edges and nodes, so I think it's clear that they refer to those. The only time I could imagine it being a problem is if the names were to conflict with those used on other internal apps (:shipit:) where it might be imported. I'd rather not use a longer BEM name with the parent included, because there are so many modifiers used on this class (see below), and it's used in so many places, so I'm trying to keep it nice and concise.

Example of the modifiers used:
```
.classed('node--data', node => node.type === 'data')
.classed('node--task', node => node.type === 'task')
.classed('node--icon', !textLabels)
.classed('node--text', textLabels)
.classed('node--active', node => node.active)
.classed('node--highlight', ifFocused(node => linkedNodes[node.id]))
.classed('node--faded', ifFocused(node => !linkedNodes[node.id]))
```

Reply:
I meant more interacting when used inside other applications, some style pollution, maybe css modules would be a way to mitigate this going forward?

Reply:
It's a possibility, and worth considering, but I reckon I'll cross that bridge when I come to it? Seems low risk for now?